portfolio/src/main/java/com/google/sps/data/WordsFrequencies.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+/**
+ * Represents frequencies of some words from music album.
+ * @author Olga Shimanskaia <olgashimanskaia@gmail.com>
+ */
+public class WordsFrequencies {
```

Comment:
Again I would add a comment to explain the purpose of this class which contains only private fields